### ISSUE TITLE
Linked content tabs with a large length mismatch requires scroll

### ISSUE THREAD

**Issue by ajh0912** [squidfunk/mkdocs-material#4342]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [X] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

When using linked content tabs with a large length disparity between each tab, the viewport remains where it was on the page, but the content ends up well beyond the current view requiring the user to scroll.

I understand that having very long content in a content tab is probably best avoided, for example using [collapsible admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#collapsible-blocks). If it's not possible to fix this, then I will use that workaround instead. I'd like to avoid that workaround if possible, as it adds extra clicks.

Possible fix, if the content would no longer be visible after switching tabs then use the content tab [Anchor links](https://squidfunk.github.io/mkdocs-material/reference/content-tabs/#anchor-links) to scroll to the tab again.

### Expected behaviour

The content from the content tab is still in view after clicking it.

### Actual behaviour

The content may be no longer visible, requiring the user to scroll to it.

### Steps to reproduce

1. Click on Tab B of Section 3
2. Section 3 is no longer visible

https://user-images.githubusercontent.com/40744246/189545745-b407309b-af9e-4eb4-89e8-55bf343eb258.mov

```
# Test

## Section 1

=== "Tab A"

    Short content 1

=== "Tab B"

    Long content 1
    ``` text
    1
    2
    3
    4
    5
    6
    7
    8
    9
    10
    11
    12
    13
    14
    15
    16
    17
    18
    19
    20
    ```

## Section 2

=== "Tab A"

    Short content 2

=== "Tab B"

    Long content 2
    ``` text
    1
    2
    3
    4
    5
    6
    7
    8
    9
    10
    11
    12
    13
    14
    15
    16
    17
    18
    19
    20
    ```

## Section 3

=== "Tab A"

    Short content 3

=== "Tab B"

    Long content 3
    ``` text
    1
    2
    3
    4
    5
    6
    7
    8
    9
    10
    11
    12
    13
    14
    15
    16
    17
    18
    19
    20
    ```

## Section 4

=== "Tab A"

    Short content 4

=== "Tab B"

    Long content 4
    ``` text
    1
    2
    3
    4
    5
    6
    7
    8
    9
    10
    11
    12
    13
    14
    15
    16
    17
    18
    19
    20
    ```
```

### Package versions

- Python: `Python 3.9.13`
- MkDocs: `mkdocs, version 1.3.0`
- Material: `Version: 8.4.3+insiders.4.22.1`

### Configuration

```yaml
site_name: Test Docs
theme:
  name: material
  features:
    - content.tabs.link
markdown_extensions:
  - admonition
  - footnotes
  - pymdownx.emoji:
      emoji_index: !!python/name:materialx.emoji.twemoji
      emoji_generator: !!python/name:materialx.emoji.to_svg
  - attr_list
  - md_in_html
  - pymdownx.superfences
  - pymdownx.tabbed:
      alternate_style: true 
  - pymdownx.details
```

### System information

- Operating system: macOS 12.4
- Browser: Firefox 104.0.1, Safari 15.5

**Comment by squidfunk** [comment]
Thanks for reporting! Fixed in ec9e2a325.

https://user-images.githubusercontent.com/932156/189924967-58c721da-54bd-4896-a044-5fd16b022e6a.mp4

**Comment by ajh0912** [comment]
Awesome, it's working perfectly now! Thanks for the quick fix.
